_includes/integrations/webcomponent-tech-ref.md

```diff
@@ -0,0 +1,590 @@
+
+To use {{site.productname}} from the {{site.cloudname}}, add the `api-key` attribute to the `tinymce-editor` element with an API from [{{site.accountpage}}]({{site.accountpageurl}}).
+
+### Configuring the editor
+
+The editor is configured by setting attributes on the `tinymce-editor` custom element.
+All attributes may be represented as strings, however some attributes will accept boolean or number inputs. Where an array, object, or function is a valid input; examples have been provided.
+
+#### Setting the initial content
+
+To set the initial editor content, add encoded HTML to the `tinymce-editor` element as content, such as:
+
+```html
+<tinymce-editor>
+  &lt;p&gt;This will be the initial content of the editor.&lt;/p&gt;
+</tinymce-editor>
+```
+
+The editor will load the text content in the `tinymce-editor` element as the initial HTML.
+
+> **Important**: The HTML must be encoded as text to prevent injected javascript from running, which may have an unintended impact on the surrounding document.
+
+#### Loading plugins
+
+To add plugins and extend the functionality of the editor, add a string of plugin identifiers using the `plugins` attribute, such as:
+
+```html
+<tinymce-editor plugins="advlist autolink link image lists charmap print preview"></tinymce-editor>
+```
+
+For a list of available plugins, see: [Add plugins to {{site.productname}}]({{site.baseurl}}/plugins/).
+
+#### Setting the editor width
+
+To set the width of the editor (content area and user interface), use the `width` attribute. For pixel-based widths, provide a number; otherwise provide a string. For example:
+
+##### Setting a pixel-based editor width
+
+```html
+<tinymce-editor width=300></tinymce-editor>
+```
+
+##### Setting a CSS unit based editor width
+
+```html
+<tinymce-editor width="50%"></tinymce-editor>
+```
+
+#### Setting the editor height
+
+To set the height of the editor (content area and user interface), use the `height` attribute. For pixel-based height, provide a number; otherwise provide a string. For example:
+
+##### Setting a pixel-based editor height
+
+```html
+<tinymce-editor height=300></tinymce-editor>
+```
+
+##### Setting a CSS unit based height
+
+```html
+<tinymce-editor height="15em"></tinymce-editor>
+```
+
+#### Setting the toolbar
+
+To set the editor toolbar buttons, use the `toolbar` attribute. For example:
+
+```html
+<tinymce-editor toolbar="undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | outdent indent"></tinymce-editor>
+```
+
+The `toolbar` attribute accepts a space-separated string of toolbar buttons with pipe characters (`|`) for grouping buttons. For a list of available toolbar buttons, see: [Toolbar Buttons Available for TinyMCE]({{site.baseurl}}/advanced/available-toolbar-buttons/).
+
+##### Disabling the toolbar
+
+To disable the toolbar, set the `toolbar` attribute to `false`. For example:
+
+```html
+<tinymce-editor toolbar=false></tinymce-editor>
```

Comment:
As before attributes should always be quoted. Attributes are always strings. Sometimes special patterns in those strings such as "false" are parsed into booleans.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1659/commits/ff98e3ed3479f619921990f2b28efb25ab1d199d